doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -270,149 +255,148 @@ automatically created.
    from the order in which they were inserted, making this implementation
    observationally different from the legacy one.
 
-.. cmd:: Hint @hint_definition : {+ @ident}
+.. _creating_hints:
 
-   The general command to add a hint to some databases :n:`{+ @ident}`.
+Creating Hints
+--------------
 
-   This command supports the :attr:`local`, :attr:`global` and :attr:`export`
-   locality attributes. When no locality is explictly given, the
+   The various `Hint` commands share these elements:
+
+   :n:`{? : {+ @ident } }` specifies the hint database(s) to add to.
+   *(Deprecated since version 8.10:* If no :token:`ident`\s
+   are given, the hint is added to the `core` database.)
+
+   Outside of sections, these commands support the :attr:`local`, :attr:`export`
+   and :attr:`global` attributes. :attr:`global` is the default.  Inside sections,
+   only the :attr:`local` attribute is supported because hints are local to sections.
+
+   When locality is specified, the
    command is :attr:`local` inside a section and :attr:`global` otherwise.
 
    + :attr:`local` hints are never visible from other modules, even if they
-     require or import the current module. Inside a section, the :attr:`local`
-     attribute is useless since hints do not survive anyway to the closure of
-     sections.
+     :cmd:`Import` or :cmd:`Require` the current module.
 
-   + :attr:`export` are visible from other modules when they import the current
-     module. Requiring it is not enough.
+   + :attr:`export` hints are visible from other modules when they :cmd:`Import` the current
+     module, but not when they only :cmd:`Require` it. This attribute is only supported for
+     the :cmd:`Hint Constructors`, :cmd:`Hint Extern`, :cmd:`Hint Immediate`, :cmd:`Hint Resolve`
+     and :cmd:`Hint Unfold` variants of the command.
```

Comment:
The set of commands supporting the attributes has just been extended so you can remove this sentence now.